src/index.ts

```diff
@@ -15,58 +15,45 @@ import { IndentationMarkerConfiguration, indentationMarkerConfig } from "./confi
 // CSS classes:
 // - .cm-indent-markers
 
-// CSS variables:
-// - --indent-marker-bg-part
-// - --indent-marker-active-bg-part
-
-/** Color of inactive indent markers. Based on RUI's var(--background-higher) */
-const MARKER_COLOR_LIGHT = '#F0F1F2';
-const MARKER_COLOR_DARK = '#2B3245';
-
-/** Color of active indent markers. Based on RUI's var(--background-highest) */
-const MARKER_COLOR_ACTIVE_LIGHT = '#E4E5E6';
-const MARKER_COLOR_ACTIVE_DARK = '#3C445C';
-
-/** Thickness of indent markers. Probably should be integer pixel values. */
-const MARKER_THICKNESS = '1px';
-
-const indentTheme = EditorView.baseTheme({
-  '&light': {
-    '--indent-marker-bg-color': MARKER_COLOR_LIGHT,
-    '--indent-marker-active-bg-color': MARKER_COLOR_ACTIVE_LIGHT
-  },
+function createIndentTheme(colors) {
```

Comment:
nit: think we can drop the `create` prefix here most extensions are functions that return extensions anyways (e.g. even `indentationMarkers` itself)

Reply:
Sounds good 👍